packages/pharos-site/src/components/Sidenav.tsx

```diff
@@ -114,9 +98,22 @@ const Sidenav: FC = () => {
               'Editing checklist',
             ].map(createSidenavLink.bind(this, 'content-style-guide'))}
           </PharosSidenavMenu>
-          <PharosSidenavMenu label="Templates"></PharosSidenavMenu>
         </PharosSidenavSection>
         <PharosSidenavSection label="Design System">
+          <PharosSidenavLink
+            href="/design-tokens"
+            isActive={window.location.pathname === withPrefix('/design-tokens')}
+            onClick={handleLinkClick}
+          >
+            Design tokens
+          </PharosSidenavLink>
+          <PharosSidenavLink
+            href="/components/textarea"
+            isActive={window.location.pathname === withPrefix('/components/textarea')}
+            onClick={handleLinkClick}
+          >
+            Text area
+          </PharosSidenavLink>
```

Comment:
```suggestion
```
Believe `Textarea` is already in the list below. I think the reason we don't see it in the expanded menu is because the list now exceeds the `50rem` max-height we capped it at. So sounds like a fix needed for the sidenav component.